[2023-03-21T23:04:50.252+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Spotify_Data_ETL.insert_date manual__2023-03-21T22:47:20.265233+00:00 [queued]>
[2023-03-21T23:04:50.258+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Spotify_Data_ETL.insert_date manual__2023-03-21T22:47:20.265233+00:00 [queued]>
[2023-03-21T23:04:50.258+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T23:04:50.259+0000] {taskinstance.py:1283} INFO - Starting attempt 13 of 15
[2023-03-21T23:04:50.259+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-21T23:04:50.266+0000] {taskinstance.py:1303} INFO - Executing <Task(SnowflakeOperator): insert_date> on 2023-03-21 22:47:20.265233+00:00
[2023-03-21T23:04:50.272+0000] {standard_task_runner.py:55} INFO - Started process 23409 to run task
[2023-03-21T23:04:50.275+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Spotify_Data_ETL', 'insert_date', 'manual__2023-03-21T22:47:20.265233+00:00', '--job-id', '2348', '--raw', '--subdir', 'DAGS_FOLDER/spotifydag.py', '--cfg-path', '/tmp/tmph_4zwc3g']
[2023-03-21T23:04:50.276+0000] {standard_task_runner.py:83} INFO - Job 2348: Subtask insert_date
[2023-03-21T23:04:50.327+0000] {task_command.py:388} INFO - Running <TaskInstance: Spotify_Data_ETL.insert_date manual__2023-03-21T22:47:20.265233+00:00 [running]> on host 1fd7b08731f2
[2023-03-21T23:04:50.372+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ananta Moharana
AIRFLOW_CTX_DAG_ID=Spotify_Data_ETL
AIRFLOW_CTX_TASK_ID=insert_date
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T22:47:20.265233+00:00
AIRFLOW_CTX_TRY_NUMBER=13
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-21T22:47:20.265233+00:00
[2023-03-21T23:04:50.373+0000] {sql.py:254} INFO - Executing: 
    INSERT INTO calendar_dim (date, month, month_name,day_of_month,day_of_week,day_name,quarter)
    VALUES ('2023-03-21', 3, 'March',21,1,'Tuesday',1)
    
[2023-03-21T23:04:50.380+0000] {base.py:73} INFO - Using connection ID 'snowflake_connection' for task execution.
[2023-03-21T23:04:50.482+0000] {base.py:73} INFO - Using connection ID 'snowflake_connection' for task execution.
[2023-03-21T23:04:50.485+0000] {connection.py:287} INFO - Snowflake Connector for Python Version: 3.0.1, Python Version: 3.7.16, Platform: Linux-5.15.49-linuxkit-aarch64-with-debian-11.6
[2023-03-21T23:04:50.486+0000] {connection.py:990} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-03-21T23:04:50.486+0000] {connection.py:1007} INFO - Setting use_openssl_only mode to False
[2023-03-21T23:04:50.997+0000] {cursor.py:738} INFO - query: [ALTER SESSION SET autocommit=False]
[2023-03-21T23:04:51.076+0000] {cursor.py:751} INFO - query execution done
[2023-03-21T23:04:51.079+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-21T23:04:51.082+0000] {sql.py:375} INFO - Running statement: INSERT INTO calendar_dim (date, month, month_name,day_of_month,day_of_week,day_name,quarter)
    VALUES ('2023-03-21', 3, 'March',21,1,'Tuesday',1), parameters: None
[2023-03-21T23:04:51.084+0000] {cursor.py:738} INFO - query: [INSERT INTO calendar_dim (date, month, month_name,day_of_month,day_of_week,day_n...]
[2023-03-21T23:04:51.530+0000] {cursor.py:751} INFO - query execution done
[2023-03-21T23:04:51.533+0000] {sql.py:384} INFO - Rows affected: 1
[2023-03-21T23:04:51.535+0000] {snowflake.py:391} INFO - Rows affected: 1
[2023-03-21T23:04:51.536+0000] {snowflake.py:392} INFO - Snowflake query id: 01ab1a88-0004-6119-0055-208700010146
[2023-03-21T23:04:51.537+0000] {cursor.py:738} INFO - query: [COMMIT]
[2023-03-21T23:04:51.839+0000] {cursor.py:751} INFO - query execution done
[2023-03-21T23:04:51.843+0000] {cursor.py:891} INFO - Number of results in first chunk: 1
[2023-03-21T23:04:51.844+0000] {connection.py:586} INFO - closed
[2023-03-21T23:04:51.885+0000] {connection.py:589} INFO - No async queries seem to be running, deleting session
[2023-03-21T23:04:52.007+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=Spotify_Data_ETL, task_id=insert_date, execution_date=20230321T224720, start_date=20230321T230450, end_date=20230321T230452
[2023-03-21T23:04:52.062+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-21T23:04:52.087+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
